RFS/src/testFixtures/java/com/rfs/framework/SearchClusterContainer.java

```diff
@@ -118,6 +119,7 @@ public enum INITIALIZATION_FLAVOR {
     }
 
     @EqualsAndHashCode
+    @ToString
```

Comment:
This is what I have out for review right now:
```suggestion
    @ToString(onlyExplicitlyIncluded = true, includeFieldNames = false)
```

Reply:
Above prettyName add `@ToString.Include` and it should do what you are looking for in an abreviated pattern